File: FC.WebMVC/Scripts/Shared/ViewModels/FestivalSearchResult.ts
module FC.Shared.ViewModels {
    import CM = FC.Core.CoreModel;
    import INT = FC.Shared.Interfaces;
    import MODELS = FC.Shared.Models;
    import MODULES = FC.Modules;
    export class FestivalSearchResult implements INT.ISearchResult {
        UmbracoID: number;
        City: string;
        Country: string;
        Name: string;
        Genres: Array<number>;
        Image: string;
        Date: Date;
        CultureStartDate: string;
        Type: string;
        Location: string;
    }
}

File: FC.Shared/ViewModels/Search/FestivalSearchResult.cs
using FC.Interfaces.ViewModels;
using FC.Shared.Entities;
using FC.Shared.EntityMapper;
using System;
using System.Collections.Generic;
using System.Globalization;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace FC.Shared.ViewModels.Search
{
    public class FestivalSearchResult : BaseModel
    {

        
        public string Country
        {
            get; set;
        }

        
        public string Location { get; set; }

        
        public DateTime Date
        {
            get; set;
        }

        
        public string City { get; set; }

        public  string CultureStartDate
        {
            set { value = Date.ToString("d", CultureInfo.CurrentCulture); }
            get { return Date.ToString("d", CultureInfo.CurrentCulture); }
        }

        
        public int[] Genres
        {
            get; set;
        }

        
        public string Image
        {
            get; set;
        }

        
        public string Name
        {
            get; set;
        }

        private string type = "festival";
        public string Type
        {
            get
            {
                return type;
            }
            set
            {
                type = value;
            }
        }
    }
}
